docs/src/Button.doc.js

```diff
@@ -218,6 +218,37 @@ function ButtonFlyoutExample() {
   />
 );
 
+card(
+  <Example
+    name="Accessibility: sequential keyboard navigation"
+    id="removeFromSequentialKeyboardNavigation"
```

Comment:
@AlbertCarreras Instead of adding a boolean prop, can we pass in `tabIndex` and set it to an enum of `0` or `-1`? Main reasons:
* It allows us to add e.g. `1` (even though it is an anti-pattern) if we do end up needing it
* Avoids boolean props: https://spicefactory.co/blog/2019/03/26/how-to-avoid-the-boolean-trap-when-designing-react-components/

Reply:
DONE